doc: updated README documentation (#531)
tl-its-umich-edu/canvas-course-manager-next: README.md

10. Go To the `Settings` option from the Admin page --> Apps --> VIew App Configuration --> +App --> Choose Configuration type `By Client_id` --> Paste the client_id from Step 8
11. Copy the Deployment_id by searching for the LTI tool you just added to the Canvas, Click setting button next it and right `click copy 'Deployment Id'`
12. Run this command `docker exec -it ccm_web python manage.py rotate_keys`. This will create the Private and Public key for LTI

> 13. Managing LTI registration to Tool: The LTI registration is completely handled from the Django Admin Console, so we need `UUID` from LTI Tool and `Client_id` and `Deployment_id` from the Canvas. So the Commandline option will help you in various use cases to set up LTI registration.  Please note *Issuer* and *Auth domain names* are different. For example, configuration in Canvas Test takes these values: `issuer=canvas.test.instructure.com` then `Auth URL=sso.test.canvaslms.com`. If you're using a dev instance, then set `issuer=canvas.dev.instructure.com` and `Auth URL=sso.dev.canvaslms.com`. More [info](https://canvas.instructure.com/doc/api/file.lti_launch_overview.html).

Reviewer: @PraveshKunwar, what we ended up using as below issuer/auth_url, and the canvas documentation link did not inform about it and hence we are adding a note here. Look at your DB table `lti_tool_ltiregistration` for your reference

issuer = canvas.instructure.com and Auth URL=sso.canvaslms.com

> 13. Managing LTI registration to Tool: The LTI registration is completely handled from the Django Admin Console, so we need `UUID` from LTI Tool and `Client_id` and `Deployment_id` from the Canvas. So the Commandline option will help you in various use cases to set up LTI registration.  Please note *Issuer* and *Auth domain names* are different. For example, configuration in Canvas Test takes these values: `issuer=canvas.test.instructure.com` then `Auth URL=sso.test.canvaslms.com`. If you're using a dev instance, then set `issuer=canvas.dev.instructure.com` and `Auth URL=sso.dev.canvaslms.com`. More [info](https://canvas.instructure.com/doc/api/file.lti_launch_overview.html).

Reviewer: ```suggestion
13. LTI registration with Canvas Dev: If you're using a Canvas dev instance (http://umich-dev.instructure.com/), then set `issuer=canvas.instructure.com` and `Auth URL=sso.canvaslms.com` when running the LTI management command .
```
